sources/packages/api/src/database/entities/disbursement-schedule.model.ts

```diff
@@ -137,4 +137,12 @@ export class DisbursementSchedule extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   studentAssessment: StudentAssessment;
+  /**
+   * Tuition Remittance Amount Requested
```

Comment:
Please convert it to an English sentence.
When possible it is also nice to add business context, for instance:
```
The tuition remittance amount is the value that is subtracted from the student grant/loan to be paid directly to the institution.
```

Reply:
Just checking, in that case should the comment in db column should also be updated?

Reply:
> Just checking, in that case should the comment in db column should also be updated?

I would prefer to keep both comments always synched unless there is a reason to not do it. For this case the DB comment is not bad, so it is not a blocker IMO.